setup.py

```diff
@@ -22,5 +22,8 @@
         ]
     },
     data_files=[('evalml/demos/data', ['evalml/demos/data/fraud_transactions.csv.tar.gz', 'evalml/demos/data/churn.csv']),
-                ("evalml/tests/data", ['evalml/tests/data/tips.csv', 'evalml/tests/data/titanic.csv'])]
+                ('evalml/tests/data', ['evalml/tests/data/tips.csv', 'evalml/tests/data/titanic.csv'])],
+    package_data = {
```

Comment:
We need to add `core-requirements.txt` to package_data so that it is included in the binary distribution and available via pip installation.

Reply:
This would install the `core-requirements.txt` file at the `site-packages` level instead of `site-packages/evalml` . It seems kind of weird that we would read a  file that is technically not in our package and it could also conflict with other files called `core-requirements.txt`. 

I think I would prefer the hardcode-a-list-of-requirements approach I think you mentioned at stand-up the other day.

![image](https://user-images.githubusercontent.com/41651716/96025109-b0a1c200-0e22-11eb-900e-ed4f819c089a.png)

Also, I think we should check if the `../` syntax works on windows?

Reply:
@angela97lin @freddyaboulton ah, I see what you're talking about. The current impl of the `evalml info` CLI happens to read in `core-requirements.txt` in order to do its thing.

I suggest we change the impl instead:
* In `cli_utils.py` delete `get_core_requirements()`
* And update `print_deps()`:
```python
def print_deps(dependencies):
    logger.info("\nINSTALLED VERSIONS")
    logger.info("------------------")
    installed_packages = get_installed_packages()
    packages_to_log = []
    for package in installed_packages:
        packages_to_log.append((x, installed_packages[x]))
        logger.info("{package}: {version}".format(package=package, version=installed_packages[package]))
```

Will that work?

Also. good point about the pathing! So I think that `data_files` and `package_data` are used during package creation, meaning they need to work on whatever platform we build source dist / wheels on. We currently do this on linux. So, we're ok for now :)

Reply:
@dsherry @freddyaboulton 

Hm, is this what we want though? By removing `get_core_requirements()` / just calling ` get_installed_packages()`, we're getting *all* of the packages the user has installed, regardless of if it is an evalml dependency or not. I'd argue then that this is not much more useful than `pip freeze`. If there's no good workaround, I'm okay with just having a hardcoded list as [Featuretools has] (https://github.com/FeatureLabs/featuretools/blob/main/featuretools/utils/cli_utils.py). Thoughts?